Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -423,6 +425,10 @@ public Item GetFocusTarget()
             if (positionOut == null) { return null; }
 
             item.SendSignal(new Signal(MathHelper.ToDegrees(targetRotation).ToString("G", CultureInfo.InvariantCulture), sender: user), positionOut);
+            if (item.Connections?.Find(c => c.Name == "distance_out") is { } distanceOut)
```

Comment:
There is no reason to find the connection like this, the signal could instead be sent using the SendSignal override that takes in the name of the connection (which is also much faster than this, because the connection is fetched from a dictionary).